CodeSignalJS/reverseInParentheses.ts

```diff
@@ -4,6 +4,42 @@
 // Ex input = "(bar)" => "rab"
 // Ex input = "foo(bar(baz))blim" => "foobazrabblim"
 
-function reverseInParentheses(s: string): string { return "string"}
+function reverseInParentheses(s: string) { 
+    let characterArray = s.split('');
+
+    const parenIndexes = (characterArray) : {start:number | undefined, end:number | undefined} => {
```

Comment:
Nice job with this logic and pulling it into a helper function is nice and clean